sui/src/config.rs

```diff
@@ -55,6 +55,19 @@ pub struct AuthorityPrivateInfo {
     pub consensus_address: SocketAddr,
 }
 
+impl Clone for AuthorityPrivateInfo {
```

Comment:
Do you only need this for tests? if so stick a #[test] attribute on it? We did a lot of work to ensure private info is not cloned :)

Reply:
Fixed